region/lowernorfair/east/Amphitheatre.json

```diff
@@ -200,13 +200,20 @@
     {
       "link": [1, 1],
       "name": "Leave With Runway",
-      "requires": [],
+      "requires": [
+        {"obstaclesCleared": ["A"]}
+      ],
       "exitCondition": {
         "leaveWithRunway": {
           "length": 1,
           "openEnd": 1
         }
-      }
+      },
+      "unlocksDoors": [
```

Comment:
At this point the door at 1 is already unlocked.
Here, we're standing on the door frame.
There could be a scenario where you door stuck to unlock the door, and that wouldn't leave 1.
As it is, the unlock requirements are the same everywhere so it's not causing an issue.